lib/models/BucketInfo.js

```diff
@@ -112,6 +117,10 @@ class BucketInfo {
         if (lifecycleConfiguration) {
             LifecycleConfiguration.validateConfig(lifecycleConfiguration);
         }
+        if (uid) {
+            assert.strictEqual(typeof uid, 'string');
+            assert.strictEqual(uid.length, 36);
```

Comment:
not sure if we should validate length of uid, but I was thinking it should be something directly from the `uuid` module

Reply:
Could be, I think it's fine to validate the length as it should be 36 (uuid length) in all cases.